src/include/migraphx/op/broadcast.hpp

```diff
@@ -68,6 +68,9 @@ struct broadcast
         {
             // the ONNX broadcast op is deprecated now, so not handling the negative
             // value of axis anymore
+            if(s0.dynamic())
+                MIGRAPHX_THROW(
+                    "BROADCAST: Single dynamic input shape not supported.  Use two inputs.");
```

Comment:
Needs test coverage for this in op_shape_test.cpp

Reply:
adding broadcast_1in_dyn_error